models/wombat/bnb/wombat_bnb_trades.sql

```diff
@@ -14,6 +14,25 @@
 
 {% set project_start_date = '2022-04-18' %}
 
+with wombat_swaps_all_pools as (
+	-- main pool
+	select * from {{ source('wombat_bnb', 'Pool_evt_Swap') }}
+	union
```

Comment:
Moreover, do you think `UNION ALL` is better in this case?
Also you can write down full used column names instead of `*`.